src/app/RecordingList/RecordingsDataTable.tsx

```diff
@@ -36,33 +36,55 @@
  * SOFTWARE.
  */
 import * as React from 'react';
-import { DataList, DataListCell, DataListCheck, DataListItem, DataListItemCells, DataListItemRow } from '@patternfly/react-core';
+import { Bullseye, DataList, DataListCell, DataListCheck, DataListItem, DataListItemCells, DataListItemRow, Spinner, Text } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
 
 export interface RecordingsDataTableProps {
   toolbar: React.ReactElement;
   tableColumns: string[];
   listTitle: string;
   isHeaderChecked: boolean;
+  isLoading: boolean;
+  isConnectionError: boolean;
   onHeaderCheck: (checked: boolean) => void;
 }
 
 export const RecordingsDataTable: React.FunctionComponent<RecordingsDataTableProps> = (props) => {
-  return (<>
-    { props.toolbar }
-    <DataList aria-label={props.listTitle}>
-      <DataListItem aria-labelledby="table-header-1">
-        <DataListItemRow>
-          <DataListCheck aria-labelledby="table-header-1" name="header-check" onChange={props.onHeaderCheck} isChecked={props.isHeaderChecked} />
-          <DataListItemCells
-            dataListCells={props.tableColumns.map((key , idx) => (
-              <DataListCell key={key}>
-                <span id={`table-header-${idx}`}>{key}</span>
-              </DataListCell>
-            ))}
-          />
-        </DataListItemRow>
-      </DataListItem>
-      { props.children }
-    </DataList>
-  </>);
+  if (props.isConnectionError) {
+    return (<>
+      <br/>
+      <Bullseye>
+        <ExclamationCircleIcon size='md' color='Red'/>
+      </Bullseye>
+      <Bullseye>
+        <Text>
+          Unable to connect
```

Comment:
I don't think such a specific error message should be used here - there could be plenty of other reasons why an error occurs, too. This should either be a generic message, or an exact message should be passed to the component.

Reply:
Ok, I changed it to display the error from the failed request